builtin/game/chatcommands.lua

```diff
@@ -148,10 +148,11 @@ core.register_chatcommand("privs", {
 	params = "<name>",
 	description = "print out privileges of player",
 	func = function(name, param)
-		param = (param ~= "" and param or name)
-		return true, "Privileges of " .. param .. ": "
+		local param = param:gsub("%s", "")
+		name = (param ~= "" and param or name)
```

Comment:
potentially misleading how name goes from caller to name of person queried.

```lua
param = (param ~= "" and param or name)
```

would maybe be better